<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authentication</class>
 <widget class="QDialog" name="Authentication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authentication</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_UsrLogin">
     <property name="minimumSize">
      <size>
       <width>182</width>
       <height>181</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_Username">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>133</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="StcUsrName">
         <property name="minimumSize">
          <size>
           <width>98</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&amp;User Name</string>
         </property>
         <property name="buddy">
          <cstring>le_Username</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="StcPWD">
         <property name="minimumSize">
          <size>
           <width>98</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&amp;Password   </string>
         </property>
         <property name="buddy">
          <cstring>LE_Pwd</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="LE_Pwd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>133</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QComboBox" name="cmb_UUTselection">
         <property name="minimumSize">
          <size>
           <width>133</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="StcPWD_2">
         <property name="minimumSize">
          <size>
           <width>98</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&amp;UUT Selection   </string>
         </property>
         <property name="buddy">
          <cstring>cmb_UUTselection</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="PB_Login">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cantarell</family>
           <pointsize>11</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;background-color: qlineargradient(spread:reflect, x1:1, y1:0.597, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 138, 0, 255));border-width: 1px;border-color: #339;border-style: solid;border-radius: 7;padding: 3px;font-size: 13px;padding-left: 5px;padding-right: 5px;font:80 11pt Cantarell;</string>
         </property>
         <property name="text">
          <string>&amp;Login</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="PB_Exit">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cantarell</family>
           <pointsize>11</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;background-color: qlineargradient(spread:reflect, x1:1, y1:0.597, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(0, 138, 0, 255));border-width: 1px;border-color: #339;border-style: solid;border-radius: 7;padding: 3px;font-size: 13px;padding-left: 5px;padding-right: 5px;font:80 11pt Cantarell;</string>
         </property>
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_Username</tabstop>
  <tabstop>LE_Pwd</tabstop>
  <tabstop>cmb_UUTselection</tabstop>
  <tabstop>PB_Login</tabstop>
  <tabstop>PB_Exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
